kong/observability/tracing/propagation/extractors/instana.lua

```diff
@@ -0,0 +1,55 @@
+local _EXTRACTOR        = require "kong.observability.tracing.propagation.extractors._base"
+local propagation_utils = require "kong.observability.tracing.propagation.utils"
+local from_hex          = propagation_utils.from_hex
+
+local INSTANA_EXTRACTOR = _EXTRACTOR:new({
+  headers_validate = {
+    any = {
+      "x-instana-t",
+      "x-instana-s",
+      "x-instana-l",
+    }
+  }
+})
+
+function INSTANA_EXTRACTOR:get_context(headers)
+
+  local trace_id_raw = headers["x-instana-t"]
+  local span_id_raw = headers["x-instana-s"]
+  local level_id_raw = headers["x-instana-l"] 
```

Comment:
the value of `headers["name"]` can be a string or a table, depending on whether the same header was passed more than once in the request, see: https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#ngxreqget_headers

We should handle the scenario where multiple headers are passed, for example by ignoring the header if passed more than once, or grabbing the first value.